### ISSUE TITLE
NavTex, Inmarsat Fleet messages should be made part of the SignalK spec

### ISSUE THREAD

**Issue by mgrouch** [SignalK/specification#616]
NavTex, Inmarsat Fleet messages should be made part of the SignalK spec

**Comment by mgrouch** [comment]
Related: https://github.com/SignalK/specification/issues/356

**Comment by htool** [comment]
Agreed. For NavTex messages are send by stations identified by StationIDs.
A message is of a certain type and carries a message id. Typically one would filter on station and then message type.

So a path with ../navtex/stationId/messageType would allow to setup listeners for only those messages that you're interested in.
